Signal/src/util/OWSBackupExportJob.m

```diff
@@ -818,16 +743,16 @@ - (BOOL)saveNextAttachmentFileToCloudWithCompletion:(OWSBackupJobCompletion)comp
         //   this record's metadata.
         // * That this record does in fact exist in our CloudKit database.
```

Comment:
Is this comment still up to date now that we don't have lastManifestItemMap?

Reply:
Fixed.